docs/user/integration/prometheus/README.md

```diff
@@ -0,0 +1,154 @@
+# Integrate with Prometheus
+
+| Category     |                                         |
+| ------------ | --------------------------------------- |
+| Signal types | metrics                                 |
+| Backend type | custom local                            |
+| OTLP-native  | yes                                     |
+
+Learn how to configure the Telemetry module to ingest metrics in a custom [Prometheus](https://prometheus.io/) instance deployed with the [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack).
+
+## Table of Content
+
+- [Integrate with Prometheus](#integrate-with-prometheus)
+  - [Table of Content](#table-of-content)
+  - [Prerequisites](#prerequisites)
+  - [Context](#context)
+
```

Comment:
One section is missing:

```suggestion
- [Procedure](#procedure)
```

If you want to go wild (and give better overview to the reader), consider adding the sub-procedures to the TOC:

```suggestion
- [Procedure](#procedure)
  - [Install the kube-prometheus-stack](#install-the-kube-prometheus-stack)
  - [Verify the kube-prometheus-stack](#verify-the-kube-prometheus-stack)
  - [Activate a MetricPipeline](#activate-a-metricpipeline)
  - [Deploy the Sample Application](#deploy-the-sample-application)
  - [Cleanup](#cleanup)
```

At which point did the user achieve the goal of the procedure? Meaning, a result like "You have successfully integrated Kyma telemetry with Prometheus."  Before or after the cleanup?

Reply:
I restructured it a bit